boskos/README.md

```diff
@@ -0,0 +1,51 @@
+# WIP boskos
+
+
+## Background
+[βοσκός](https://en.wiktionary.org/wiki/%CE%B2%CE%BF%CF%83%CE%BA%CF%8C%CF%82) - cattle-master in greek!
+
+boskos is a http service that allow k8s test jobs to run independently from GCP projects.
+
+## Introduction
+
+Imagine boskos is a ranch with cattle, and to perform each job in the ranch you need to hold a cattle. 
+
+The ranch is inited with a list of GCP projects, one JSON entry per line, from `-config` flag
+
+Interface:
+
+* `/request?job=FOO&duration=X`: with job FOO, checkout a cattle for duration X.
+X can be in any unit accept by [`time.ParseDuration`](https://golang.org/pkg/time/#ParseDuration)
+* `/return?job=FOO`: Job FOO is done, return the cattle checked out by job FOO.
```

Comment:
We should not assume that there is only one instance of the job running at the same time.

There could be a zombie job. We could update to run concurrent instances of the job